.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node: ['15', '14', '12']
+        # this is needed only for some plugins
+        # but I don't see how we can disable it without having to remove it when we don't need it
+        hapi: ['20', '19']
```

Comment:
Why include this here? It is not a plugin, and has no dependency on hapi.

Reply:
This was just some kind of brain dump to make an example for others in order to match every configuration we had on Travis. I'll remove it now.